### ISSUE TITLE
Design Intent - Accessibility Feedback - Octonaut, PAWS Program, PAWS Program

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#84668]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

-

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** As much as possible, provide users with concrete information on who they should contact/what they should be asking for when contacting a participating facility, eg. "Contact X office at the VAMC and ask about PAWS." One of the symptoms of PTSD can be a loss of ability to focus and a general feeling of disconnectedness or lack of interest. If someone decides they're interested in the program, you want to give them a well-defined path for engaging with the program. A higher cognitive load (figuring out what they're supposed to do next) can become a barrier.
- [ ] **Should:** As much as possible, provide users who are likely ineligible with some explanation of why you've reached that conclusion. That content may be duplicative of the eligibility information already on the page, but I think that's preferable to someone thinking they should be eligible and feeling like there's no clear explanation for why they're being told they aren't. Providing an explanation gives you something you can use to follow up as needed, eg. talking to your doctor to say "This says I'm not eligible because X, but didn't you tell me...?" in order to resolve some kind of misunderstanding.
- [ ] **Should:** Noting a mismatch between the wireframe and the user flow -- the user flow has a "Click check eligibility button" step after login, and the wireframes seem to give the eligibility response immediately after login. I like not having to click the button! It's not necessarily an accessibility problem to have that extra button click, but if it's a step we can remove that's a little bit of friction smoothed out.
- [ ] **Engineering note:** This is down the road a bit, but you'll want to set keyboard focus on the eligibility response alert after login. The biggest benefit here is for screen reader users (but also low vision users, keyboard-only users, and a few other groups). Rather than having to re-navigate to the section of the page you left off at when you authenticated, just go right back to that spot.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone
